lessons/lesson2.md

```diff
@@ -1,129 +1,30 @@
-## Lesson 2: Write your app
+## Lesson 2: Debugging Application Code
 
-# H1
-## H2
-### H3
-#### H4
-##### H5
-###### H6
+To overcome the error that you got in the previous lesson, now open your app inside Adobe Experience Cloud (ExC) Shell on the same Chrome Debug window: https://experience.adobe.com/?devMode=true#/apps/?localDevUrl=https://localhost:9080. You might be asked to log in using your Adobe ID. When opened, the UI is almost similar to that when you access it on localhost, except the ExC Shell on top.  
```

Comment:
Any particular reason for opening it in ExC Shell instead of https://localhost:9080 ?